app/views/identities/_show_sites.erb

```diff
@@ -18,3 +18,5 @@
     <% end %>
   <% end %>
 </ul>
+<br>
+<%= link_to "Enable/Disable Portal", toggle_portal_identity_path(@identity.uid, portal: "portal"), class: "btn btn-primary", method: :post if current_user.user_actions&.dig("admin", "can_administer") %>
```

Comment:
should the parameter be called 'portal'? or should it be more generic, like "feature"?
In the UI when I see this button, it will say "Enable/Disable Portal" - how will I know whether the toggle is on or off?

Reply:
Cheers! I will try to implement different buttons for relevant statuses of portal, but was not sure if it was worth pinging DB everytime. Status of the feature can be checked via Flipper built-in UI, but not ideal as I see it

Reply:
done 😃 
<img width="355" alt="image" src="https://user-images.githubusercontent.com/70934030/121873752-09c34c00-cd4a-11eb-9bd8-3edf75925b92.png">
<img width="529" alt="image" src="https://user-images.githubusercontent.com/70934030/121873792-147de100-cd4a-11eb-8280-bdf21bed94f6.png">